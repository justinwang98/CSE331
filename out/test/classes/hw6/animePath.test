# Test the path on a graph that contains several nodes

LoadGraph anime anime.tsv
FindPath anime Asuna Zoro