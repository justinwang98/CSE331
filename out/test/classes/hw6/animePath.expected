# Test the path on a graph that contains several nodes

loaded graph anime
path from Asuna to Zoro:
Asuna to Kirito via SAO
Kirito to Zoro via OnePiece